sda-commons-server-trace/build.gradle

```diff
@@ -1,6 +1,7 @@
 dependencies {
   api project(':sda-commons-server-dropwizard')
   api project(':sda-commons-shared-tracing')
+  api project (':sda-commons-server-opentelemetry')
```

Comment:
We should not spread this dependency into more modules, see https://github.com/SDA-SE/sda-dropwizard-commons/pull/2248